frontend/src/views/Layout/ContentFooter.vue

```diff
@@ -23,30 +23,33 @@
         </b-nav>
       </b-col>
     </b-row>
-      <br>
-     <br>
-   <b-row >
-            <b-col class="nav-link text-center"  @click.prevent="setLocale('en')">{{ $t('languages.en') }}</b-col>
-            <b-col class="nav-link text-center" @click.prevent="setLocale('de')">{{ $t('languages.de') }}</b-col>
-        </b-row>
+    <br>
+    <br>
+    <b-row >
+      <b-col class="nav-link text-center"  @click.prevent="setLocale('en')">{{ $t('languages.en') }}XXXXX</b-col>
+      <b-col class="nav-link text-center" @click.prevent="setLocale('de')">{{ $t('languages.de') }}</b-col>
```

Comment:
```suggestion
      <b-col class="nav-link text-center"  @click.prevent="setLocale('en')">{{ $t('languages.en') }}</b-col>
      <b-col class="nav-link text-center" @click.prevent="setLocale('de')">{{ $t('languages.de') }}</b-col>
```

What are the Xses for?

Reply:
Ups.